fhir-persistence-blob-app/README.md

```diff
@@ -0,0 +1,258 @@
+# Payload Offload for Azure Blob
+
+The IBM FHIR Server can be configured to store the resource payload records in an Azure Blob container. Each record is stored as a JSON string with UTF-8 encoding. The IBM FHIR Server relies on the blob service to compress/encrypt the data at rest. The blob service must apply the necessary security controls required when storing PHI, and connection to the service must use an encrypted transport.
+
+## Configuring the IBM FHIR Server to use Payload Offloading with Azure Blob
+
+To enable payload offloading using Azure Blob storage, complete the steps summarized below:
+
+1. Pick a container name to use for each tenant/datasource combination;
+2. Create a database.properties file with connection information for the FHIRSERVER user (not FHIRADMIN);
+3. Configure payload offloading in the main `default/fhir-server-config.json` file;
+4. Configure payload offloading in each tenant-specific `fhir-server-config.json` file;
+5. Create the container using the Azure Blob user interface, the Azure Blob API or the IBM FHIR Server fhir-persistence-blob-app-*-cli.jar tool;
+6. (Optional) Periodically run the reconciliation tool to remove any orphan resource records from the offload datastore.
+
+Take the following restrictions into account:
+
+1. Payload offloading is enabled at the server level. Offloading cannot be enabled/disabled on a per-tenant basis. If you want to support offloading for just one tenant, use a different IBM FHIR Server instance;
+2. Payload offloading must be configured prior to ingesting any resource data;
+3. Payload offloading must not be disabled after resource data has been ingested;
+4. When payload offloading is enabled, FHIR resources are limited to 4 MiB in size when rendered as uncompressed JSON. If this limit is too small, please raise an issue in the IBM FHIR Server [repository](https://github.com/IBM/FHIR/issues) with a description of your use case.
```

Comment:
I'm thinking we should have an open issue to track this, even if we decide not to support it for our 5.0

Reply:
issue #3414